NeuroID/src/main/java/com/neuroid/tracker/storage/NIDSharedPrefsDefaults.kt

```diff
@@ -47,6 +47,16 @@ class NIDSharedPrefsDefaults(
         putString(NID_UID, userId)
     }
 
+    fun setRegisteredUserID(userId: String) {
+        putString(NID_REG_UID, userId)
+    }
+
+    fun getRegisteredUserId(): Any {
```

Comment:
same as above

Reply:
These can stay the same and follow the existing convention in the file.

Reply:
fixed. thanks!